File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/src/OpsTrackingModel/UserRoleView.java
package OpsTrackingModel;

//import com.gigaspaces.annotation.pojo.SpaceClass;
//import com.gigaspaces.annotation.pojo.SpaceProperty;

import java.io.Serializable;

//@SpaceClass
public class UserRoleView implements Serializable {
    /**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String _userId = null;
    private String _roleCode = null;
    private String _descr = null;


  //  @SpaceProperty(index = SpaceProperty.IndexType.BASIC)
    public String get_userId() {
        return _userId;
    }

    public void set_userId(String _userId) {
        this._userId = _userId;
    }

    public String get_roleCode() {
        return _roleCode;
    }

    public void set_roleCode(String _roleCode) {
        this._roleCode = _roleCode;
    }

    public String get_descr() {
        return _descr;
    }

    public void set_descr(String _activeFlag) {
        this._descr = _activeFlag;
    }
}

File: DotNet/Apps/ConfMgrUserRolesGUI/DBAccess.SqlServer/UserRoleView.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpsTrackingModel
{
    [System.Serializable]
    public class UserRoleView
    {
        private string _userId = null;
        private string _roleCode = null;
        private string _descr = null;

        public string UserId
        {
            get { return _userId; }
            set { _userId = value; }
        }

        public string RoleCode
        {
            get { return _roleCode; }
            set { _roleCode = value; }
        }

        public string Descr
        {
            get { return _descr; }
            set { _descr = value; }
        }
    }
}
